src/track/public-types.ts

```diff
@@ -22,8 +22,8 @@ export interface Package {
 }
 
 export interface Shipment {
-  shipmentID?: string;
-  carrierID?: string;
+  shipmentId?: string;
```

Comment:
Based on the discussions in [slack](https://auctane.slack.com/archives/C01G1SBEC1W/p1624475161002500)